.github/workflows/test_in_devenv.yml

```diff
@@ -84,6 +89,48 @@ jobs:
               exit 1
             fi
 
+      - name: Test CUDA Quantum MPI4Py Plugin
+        uses: ./.github/actions/run-in-docker
+        with:
+          image: cuda-quantum-dev:local
+          shell: bash
+          run: |
+            cd $CUDAQ_REPO_ROOT
+            python3 -m pip install mpi4py~=3.1
+            rm -f build/lib/plugins/libcudaq-comm-plugin.so
+            ctest --test-dir build -R MPIApiTest -V
+            mpi4py_status=$?   
+            if [ ! $mpi4py_status -eq 0 ] ; then
+              echo "::error file=test_in_devenv.yml::Test CUDA Quantum MPI4Py Plugin failed with status $mpi4py_status."
+              exit 1
+            fi
+      
+      - name: Test CUDA Quantum MPI Plugin Activation
+        uses: ./.github/actions/run-in-docker
+        with:
+          image: cuda-quantum-dev:local
+          shell: bash
+          run: |
+            cd /usr/local/cudaq/distributed_interfaces/ 
+            # Set MPI_PATH depending on OMPI/MPICH
+            has_ompiinfo=$(which ompi_info || true)
+            if [[ ! -z $has_ompiinfo ]]; then
+              export MPI_PATH="/usr/lib/$(uname -m)-linux-gnu/openmpi/"
```

Comment:
The MPI_PATH should be set when installing MPI, rather than here. I.e. it should be set in cudaq.dev.Dockerfile if Arg mpi is defined.

Reply:
This seems to be difficult in this case due to the `$(uname -m)` part, namely docker doesn't allow setting `ENV` by the output of a shell command. Do you know a trick to achieve this :-) ?

I've seen another comment (which I couldn't find anymore) about checking that no `MPI_PATH` should be set when `mpi` argument is set. This has been implemented in https://github.com/NVIDIA/cuda-quantum/pull/966/commits/4caae515d251eddea52236a43c07920a01d62e4d#diff-6ad23156dcc4425b23c269621443ddcb34e370f7ebf81db49b4c1b732301caed

Reply:
Ech, I forgot that the buildkit does not define a variable for that (e.g. the TARGETARCH amd64 is accessible automatically in a multi-platform build, but it stops there - there is no way to extract x86-64 from that unless you pass it manually as argument). Alright, for testing that is fine.